flow/e2e/clickhouse/peer_flow_ch_test.go

```diff
@@ -776,3 +776,53 @@ func (s ClickHouseSuite) Test_Types_CH() {
 	env.Cancel()
 	e2e.RequireEnvCanceled(s.t, env)
 }
+
+func (s ClickHouseSuite) Test_Unprivileged_Postgres_Columns() {
+	// This test is for checking that the flow can handle a table with columns that the user does not have SELECT privileges on
+	srcTableName := "test_unprivileged_columns"
+	srcFullName := s.attachSchemaSuffix(srcTableName)
+	dstTableName := "test_unprivileged_columns_dst"
+
+	_, err := s.Conn().Exec(context.Background(), fmt.Sprintf(`
+		CREATE TABLE IF NOT EXISTS %s (
```

Comment:
should include some oddly named columns (apostrophes etc), including in the set of excluded columns